playwright.config.ts

```diff
@@ -21,16 +21,27 @@ const config: PlaywrightTestConfig = {
   workers: process.env.CI ? 1 : undefined,
   testDir: 'tests',
   timeout: 120 * 1000,
+  reporter: process.env.CI ? 'blob' : 'dot',
   projects: [
     {
       name: 'chromium',
+      grep: /webauth/,
```

Comment:
set it so the tests with the word `webauthn` only run on chromium.

We can take advantage of some playwright features like this further if we want. you can @scope tests and configure them like so